indexers/vector/ScannIndexer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jinaai/jina
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN pip install -r requirements.txt
```

Comment:
didn't u have so many problems installing scann? Is it just enough to do `pip install scann`? Have you tested this build?

Reply:
I was just updating the README of this. 
`pip install scann` This works for Linux, otherwise you need to install from source, I updated the README with this

Reply:
ok perfect then